test/powershell/Modules/Microsoft.PowerShell.Utility/Update-FormatData.Tests.ps1

```diff
@@ -57,3 +57,50 @@ Describe "Update-FormatData basic functionality" -Tags "CI" {
         { Update-FormatData -Prepend $testfile -WhatIf } | Should Not Throw
 	}
 }
+
+
+Describe "Update-FormatData with resources in CustomControls" -Tags "CI" {
+
+    BeforeAll {
+        $templatePath = Join-Path $PSScriptRoot 'FileInfo.format.ps1xml'
+        $formatFilePath = Join-Path $TestDrive 'FileInfo.format.ps1xml'
+        $ps = [powershell]::Create()
+        $iss = [system.management.automation.runspaces.initialsessionstate]::CreateDefault2()
+        $rs = [system.management.automation.runspaces.runspacefactory]::CreateRunspace($iss)
+        $rs.Open()
+        $ps.Runspace = $rs
+    }
+    AfterAll {
+        $rs.Close()
+        $ps.Dispose()
+    }
+    Context "Validate Update-FormatData" {
+	    It "Resources in WindowsPS syntax should be loaded successfully" {
+			$format = Get-Content -Path $templatePath -Raw
+			$format.Replace("%BaseName%","FileSystemProviderStrings") | Set-Content -Path $formatFilePath -Force
+			$null = $ps.AddScript("Update-FormatData -PrependPath $formatFilePath")
+			$ps.Streams.Error.Clear()
+            $ps.Invoke()
+            $ps.HadErrors | Should be $false
+
+	    }
+		It "Resources in CorePS syntax should be loaded successfully" {
+			$format = Get-Content -Path $templatePath -Raw
+			$format.Replace("%BaseName%","System.Management.Automation.resources.FileSystemProviderStrings") | Set-Content -Path $formatFilePath -Force
+			$null = $ps.AddScript("Update-FormatData -PrependPath $formatFilePath")
+			$ps.Streams.Error.Clear()
+            $ps.Invoke()
+            $ps.HadErrors | Should be $false
+	    }
+		It "Verify assembly path in error message when resource is Not found" {
+			$format = Get-Content -Path $templatePath -Raw
+			$format.Replace("%BaseName%","NonExistingResource") | Set-Content -Path $formatFilePath -Force
+			$null = $ps.AddScript("Update-FormatData -PrependPath $formatFilePath")
+			$ps.Streams.Error.Clear()
+            $ps.Invoke()
+			$sma = [appdomain]::CurrentDomain.GetAssemblies() | ? { if ($_.Location) {$_.Location.EndsWith("System.Management.Automation.dll")}}
+			$smaLocation = $sma.Location
+			$ps.Streams.Error | %{ $_.Exception.Message.Contains($smaLocation) | Should be $true }

```

Comment:
this fails with
`parsing 'C:\Users\GitHub\PowerShell\New\src\powershell-win-core\bin\Debug\netcoreapp2.0\win10-x64\publish\System.Management.Automation.dll' - Unrecognized escape sequence \\U.`